framework/src/modules/interoperability/mainchain/cc_commands/registration.ts

```diff
@@ -38,7 +36,7 @@ export class MainchainCCRegistrationCommand extends BaseInteroperabilityCCComman
 			registrationCCMParamsSchema,
 			ccm.params,
 		);
-		const interoperabilityStore = this._getInteroperabilityStore(ctx.getStore);
+		const interoperabilityStore = this.getInteroperabilityStore(ctx.getStore);
```

Comment:
I am not sure if using an instance of `SidechainInteroperabilityStore` for mainchain commands is a good idea or maybe I am missing something ?

Reply:
Ishan hurried to merge this PR late at night because it's blocking and mixed with tiredness he got confused 🙂But I agree with Rishi, with base class implementation this call here is using a sidechain interoperability store, just to clarify for world wide web.

Reply:
same as above comment